src/main/java/pages/openforms/OpenFormsPage.java

```diff
@@ -5,22 +5,30 @@
 import com.microsoft.playwright.options.AriaRole;
 
 public class OpenFormsPage {
+    public final Locator inloggenDigidButton;
+    public final Locator uitloggenDigidButton;
+    public final Locator aanvraagZonderDigidButton;
 
+    public final Locator cookieButton;
     public final Locator buttonAccepteerCookies;
+
     public final Locator linkActiveStep;
     public final Locator textlabelHeaderH1;
     public final Locator textlabelHeaderH2;
-    public final Locator inloggenDigidButton;
-    public final Locator cookieButton;
     public final Locator headerFirstFormStep;
 
+
     public OpenFormsPage(Page page) {
+        this.inloggenDigidButton = page.locator("//button[text()='Inloggen']");
```

Comment:
Als het buttons zijn is het mooier om de page.getByRole(AriaRole.BUTTON, new Page.GetByRoleOptions().setName("Inloggen")  te gebruiken.

Reply:
Ik heb de pattern van generiekeOpenformsPage gevolgd, die alleen maar xpaths gebruiken. 
Houden we eenzelfde stijl aan(xpaths) of willen we dit gaan mixen met wat er in de docs beschikbaar is ?

Reply:
ik heb het aangepast. Ik zie dat er meerdere soorten elementstijlen zijn gebruikt over de framework heen.

Reply:
De best practice van Playwright is om de page.getByRole te gebruiken als het kan. Net als page.getByText etc. Dit zijn namelijk allemaal methodes zoals de gebruiker de UI gebruikt (https://playwright.dev/docs/best-practices). De testen zijn 1 op 1 overgezet vanuit Selenium dus er zullen nog een heleboel xpaden gebruikt worden waar het misschien anders kan. Dus als we dit tegenkomen dan kunnen we het aanpassen

Reply:
ok top! mee eens :) Ik heb het gepushed